fe/fe-core/src/main/java/com/starrocks/sql/parser/SqlParser.java

```diff
@@ -2,28 +2,66 @@
 package com.starrocks.sql.parser;
 
 import com.google.common.collect.Lists;
+import com.starrocks.analysis.InsertStmt;
+import com.starrocks.analysis.SqlScanner;
 import com.starrocks.analysis.StatementBase;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.util.SqlParserUtils;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.sql.ast.QueryStatement;
 import org.antlr.v4.runtime.CharStreams;
 import org.antlr.v4.runtime.CommonTokenStream;
 
+import java.io.StringReader;
 import java.util.List;
 
 public class SqlParser {
-    public static List<StatementBase> parse(String sql) {
-        StarRocksLexer lexer = new StarRocksLexer(new CaseInsensitiveStream(CharStreams.fromString(sql)));
-        CommonTokenStream tokenStream = new CommonTokenStream(lexer);
-        StarRocksParser parser = new StarRocksParser(tokenStream);
-        parser.removeErrorListeners();
-        parser.addErrorListener(new ErrorHandler());
-
-        StarRocksParser.SqlStatementsContext sqlStatementsContext = parser.sqlStatements();
-
-        AstBuilder astBuilder = new AstBuilder();
+    public static List<StatementBase> parse(String sql, ConnectContext session) {
         List<StatementBase> statements = Lists.newArrayList();
-        for (int i = 0; i < sqlStatementsContext.singleStatement().size(); ++i) {
-            StarRocksParser.SingleStatementContext singleStatementContext = sqlStatementsContext.singleStatement(i);
-            statements.add((StatementBase) astBuilder.visitSingleStatement(singleStatementContext));
+        String[] splitSql = sql.split(";");
```

Comment:
What if the input string contains only `;`, like `;;;;;;;` ?

Reply:
It's really not a good way to write here. On the one hand, the g4 grammar file already contains the logic of multiple statement splitting, so the processing will be more elegant. But on the other hand, the current incomplete Parser refactoring may bring a problem, that is, multiple statements may contain some states that can be parsed and others that cannot be parsed. Therefore, it needs to be divided into separate statements and parsed independently.
On the other hand, ";;;;" will be parsed as multiple empty sql processing, and there is no problem of mutual influence

Reply:
Record an issue and fix the bug later @HangyuanLiu